### PR TITLE
migration-tools: avoid wide table

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

- avoid wide table
- `lighting` -> `lightning`
- Fix https://github.com/pingcap/docs-cn/issues/13392

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/9580
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
Removed the **needs-cherry-pick-release-6.3** label because the v6.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.3 and will no longer receive new updates.

File: migration-tools.md [hunk 1]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
-| [TiCDC](/ticdc/ticdc-overview.md)| 通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，具有将数据还原到与上游任意 TSO 一致状态的能力，支持其他系统订阅数据变更。|TiDB | TiDB，MySQL，Kafka，Confluent|提供开放数据协议 (TiCDC Open Protocol)。 | TiCDC 只能同步至少存在一个有效索引的表。暂不支持以下场景：暂不支持单独使用 RawKV 的 TiKV 集群。暂不支持在 TiDB 中创建 SEQUENCE 的 DDL 操作和 SEQUENCE 函数。|
-|[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) | 用于 TiDB 集群间的增量数据同步，如将其中一个 TiDB 集群作为另一个 TiDB 集群的从集群。| TiDB | TiDB，MySQL，Kafka，增量备份文件|支持实时备份和恢复。备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复。 |与部分 TiDB 版本不兼容，不能一起使用。推荐使用 [TiCDC](/ticdc/ticdc-overview.md) 替代 TiDB Binlog。|
-|[sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md) | 用于校验 MySQL/TiDB 中两份数据的一致性。|TiDB，MySQL | TiDB，MySQL| 提供了修复数据的功能，适用于修复少量不一致的数据。|对于 MySQL 和 TiDB 之间的数据同步不支持在线校验。不支持 JSON、BIT、BINARY、BLOB 等类型的数据。 |
+本文介绍了使用这些工具的场景、支持的上下游、优势和相关限制等信息。请根据你的需求选择合适的工具。
+
+## [TiDB Data Migration (DM)](/dm/dm-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于将数据从与 MySQL 协议兼容的数据库迁移到 TiDB。</span> |
```
Comment (qiancai) [inline migration-tools.md:14]:
```suggestion
| 使用场景 | <span style="font-weight:normal">用于将数据从与 MySQL 协议兼容的数据库迁移到 TiDB</span> |
```

File: migration-tools.md [hunk 2]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+|---|---|
+| **上游** | MySQL，MariaDB，Aurora，MySQL |
+| **下游** | TiDB |
+| **主要优势** | <ul><li>一体化的数据迁移任务管理工具，支持全量迁移和增量同步</li><li>支持对表与操作进行过滤</li><li>支持分库分表的合并迁移</li></ul>|
+| **使用限制** | 数据导入速度与 TiDB Lightning 的 [Logical Import Mode](/tidb-lightning/tidb-lightning-logical-import-mode.md) 大致相同，而比TiDB Lightning 的 [Physical Import Mode](/tidb-lightning/tidb-lightning-physical-import-mode.md) 低很多。建议用于 1 TB 以内的存量数据迁移。 |
+
+## [Dumpling](/dumpling-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于将数据从 MySQL/TiDB 进行全量导出。</span> |
```
Comment (qiancai) [inline migration-tools.md:23]:
```suggestion
| 使用场景 | <span style="font-weight:normal">用于将数据从 MySQL/TiDB 进行全量导出</span> |
```

File: migration-tools.md [hunk 3]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+|---|---|
+| **上游** | MySQL，TiDB |
+| **下游** | SQL，CSV |
+| **主要优势** | <ul><li>支持全新的 table-filter，筛选数据更加方便</li><li>支持导出到 Amazon S3 云盘</li></ul> |
+| **使用限制** | <ul><li>如果导出后计划往非 TiDB 的数据库恢复，建议使用 Dumpling。</li><li>如果导出后计划往另一个 TiDB 恢复，建议使用 [BR](/br/backup-and-restore-tool.md)。 </li></ul> |
+
+## [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于将数据全量导入到 TiDB。</span> |
```
Comment (qiancai) [inline migration-tools.md:32]:
```suggestion
| 使用场景 | <span style="font-weight:normal">用于将数据全量导入到 TiDB</span> |
```

File: migration-tools.md [hunk 4]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+|---|---|
+| **上游（或输入源文件）** | <ul><li>Dumpling 输出的文件</li><li>CSV 文件</li><li>从 Hive 或者 Aurora 导出的 Parquet 文件</li><li>从本地盘或 Amazon S3 云盘读取数据</li></ul>|
+| **下游** | TiDB |
+| **主要优势** | <ul><li>支持快速导入大量数据，实现快速初始化 TiDB 集群的指定表</li><li>支持断点续传</li><li>支持数据过滤</li></ul> |
+| **使用限制** | <ul><li>如果使用 [Physical Import Mode](/tidb-lightning/tidb-lightning-physical-import-mode.md) 进行数据导入，TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。</li><li>如果你不希望 TiDB 集群的对外服务受到影响，可以参考 TiDB Lightning [Logical Import Mode](/tidb-lightning/tidb-lightning-logical-import-mode.md) 中的硬件需求与部署方式进行数据导入。</li></ul> |
+
+## [Backup & Restore (BR)](/br/backup-and-restore-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">通过对大数据量的 TiDB 集群进行数据备份和恢复，实现数据迁移。</span> |
```
Comment (qiancai) [inline migration-tools.md:41]:
```suggestion
| 使用场景 | <span style="font-weight:normal">通过对大数据量的 TiDB 集群进行数据备份和恢复，实现数据迁移</span> |
```

File: migration-tools.md [hunk 5]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+|---|---|
+| **上游** | TiDB |
+| **下游（或输出文件）** | SST，backup.meta 文件，backup.lock 文件 |
+| **主要优势** | <ul><li>适用于向另一个 TiDB 迁移数据。</li><li>支持数据冷备份到外部存储，可以用于灾备恢复。</li></ul> |
+| **使用限制** | <ul><li>BR 恢复到 TiCDC / Drainer 的上游集群时，恢复数据无法由 TiCDC / Drainer 同步到下游。</li><li>BR 只支持在 `new_collations_enabled_on_first_bootstrap` 开关值相同的集群之间进行操作。</li></ul> |
+
+## [TiCDC](/ticdc/ticdc-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，具有将数据还原到与上游任意 TSO 一致状态的能力，支持其他系统订阅数据变更。</span> |
```
Comment (qiancai) [inline migration-tools.md:50]:
```suggestion
| 使用场景 | <span style="font-weight:normal">通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，具有将数据还原到与上游任意 TSO 一致状态的能力，支持其他系统订阅数据变更</span> |
```

File: migration-tools.md [hunk 6]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+|---|---|
+| **上游** | TiDB |
+| **下游** | TiDB，MySQL，Kafka，Confluent |
+| **主要优势** | 提供开放数据协议 (TiCDC Open Protocol)。|
+| **使用限制** | TiCDC 只能同步至少存在一个有效索引的表。暂不支持以下场景：<ul><li>单独使用 RawKV 的 TiKV 集群。</li><li>在 TiDB 中创建 SEQUENCE 的 DDL 操作和 SEQUENCE 函数。</li></ul> |
+
+## [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于 TiDB 集群间的增量数据同步，如将其中一个 TiDB 集群作为另一个 TiDB 集群的从集群。</span> |
```
Comment (qiancai) [inline migration-tools.md:59]:
```suggestion
| 使用场景 | <span style="font-weight:normal">用于 TiDB 集群间的增量数据同步，如将其中一个 TiDB 集群作为另一个 TiDB 集群的从集群</span> |
```

File: migration-tools.md [hunk 7]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+|---|---|
+| **上游** | TiDB |
+| **下游（或输出文件）** | TiDB，MySQL，Kafka，增量备份文件 |
+| **主要优势** | <ul><li>支持实时备份和恢复。</li><li>备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复。</li></ul> |
+| **使用限制** | 与部分 TiDB 版本不兼容，不能一起使用。推荐使用 [TiCDC](/ticdc/ticdc-overview.md) 替代 TiDB Binlog。|
+
+## [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于校验 MySQL/TiDB 中两份数据的一致性。</span> |
```
Comment (qiancai) [inline migration-tools.md:68]:
```suggestion
| 使用场景 | <span style="font-weight:normal">用于校验 MySQL/TiDB 中两份数据的一致性</span> |
```

File: migration-tools.md (hunk 1 above)
Comment (qiancai) [reply migration-tools.md:14]: 标题行里可以不用句号结尾

File: migration-tools.md [hunk 8]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
-| [TiCDC](/ticdc/ticdc-overview.md)| 通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，具有将数据还原到与上游任意 TSO 一致状态的能力，支持其他系统订阅数据变更。|TiDB | TiDB，MySQL，Kafka，Confluent|提供开放数据协议 (TiCDC Open Protocol)。 | TiCDC 只能同步至少存在一个有效索引的表。暂不支持以下场景：暂不支持单独使用 RawKV 的 TiKV 集群。暂不支持在 TiDB 中创建 SEQUENCE 的 DDL 操作和 SEQUENCE 函数。|
-|[TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) | 用于 TiDB 集群间的增量数据同步，如将其中一个 TiDB 集群作为另一个 TiDB 集群的从集群。| TiDB | TiDB，MySQL，Kafka，增量备份文件|支持实时备份和恢复。备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复。 |与部分 TiDB 版本不兼容，不能一起使用。推荐使用 [TiCDC](/ticdc/ticdc-overview.md) 替代 TiDB Binlog。|
-|[sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md) | 用于校验 MySQL/TiDB 中两份数据的一致性。|TiDB，MySQL | TiDB，MySQL| 提供了修复数据的功能，适用于修复少量不一致的数据。|对于 MySQL 和 TiDB 之间的数据同步不支持在线校验。不支持 JSON、BIT、BINARY、BLOB 等类型的数据。 |
+本文介绍了使用这些工具的场景、支持的上下游、优势和相关限制等信息。请根据你的需求选择合适的工具。
+
+## [TiDB Data Migration (DM)](/dm/dm-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于将数据从与 MySQL 协议兼容的数据库迁移到 TiDB。</span> |
+|---|---|
+| **上游** | MySQL，MariaDB，Aurora，MySQL |
```
Comment (Oreoxmt) [inline migration-tools.md:16]:
```suggestion
| **上游** | MySQL，MariaDB，Aurora |
```

File: migration-tools.md [hunk 9]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+| **下游** | TiDB |
+| **主要优势** | <ul><li>一体化的数据迁移任务管理工具，支持全量迁移和增量同步</li><li>支持对表与操作进行过滤</li><li>支持分库分表的合并迁移</li></ul>|
+| **使用限制** | 数据导入速度与 TiDB Lightning 的 [Logical Import Mode](/tidb-lightning/tidb-lightning-logical-import-mode.md) 大致相同，而比TiDB Lightning 的 [Physical Import Mode](/tidb-lightning/tidb-lightning-physical-import-mode.md) 低很多。建议用于 1 TB 以内的存量数据迁移。 |
+
+## [Dumpling](/dumpling-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于将数据从 MySQL/TiDB 进行全量导出。</span> |
+|---|---|
+| **上游** | MySQL，TiDB |
+| **下游** | SQL，CSV |
```
Comment (qiancai) [inline migration-tools.md:26]:
```suggestion
| **下游（输出文件）** | SQL，CSV |
```

File: migration-tools.md [hunk 10]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+| **主要优势** | <ul><li>支持全新的 table-filter，筛选数据更加方便</li><li>支持导出到 Amazon S3 云盘</li></ul> |
+| **使用限制** | <ul><li>如果导出后计划往非 TiDB 的数据库恢复，建议使用 Dumpling。</li><li>如果导出后计划往另一个 TiDB 恢复，建议使用 [BR](/br/backup-and-restore-tool.md)。 </li></ul> |
+
+## [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于将数据全量导入到 TiDB。</span> |
+|---|---|
+| **上游（或输入源文件）** | <ul><li>Dumpling 输出的文件</li><li>CSV 文件</li><li>从 Hive 或者 Aurora 导出的 Parquet 文件</li><li>从本地盘或 Amazon S3 云盘读取数据</li></ul>|
```
Comment (qiancai) [inline migration-tools.md:34]:
```suggestion
| **上游（输入源文件）** | <ul><li>Dumpling 输出的文件</li><li>从 Amazon Aurora 或 Apache Hive 导出的 Parquet 文件</li><li>CSV 文件</li><li>从本地盘或 Amazon S3 云盘读取数据</li></ul>|
```

File: migration-tools.md [hunk 11]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+| **下游** | TiDB |
+| **主要优势** | <ul><li>支持快速导入大量数据，实现快速初始化 TiDB 集群的指定表</li><li>支持断点续传</li><li>支持数据过滤</li></ul> |
+| **使用限制** | <ul><li>如果使用 [Physical Import Mode](/tidb-lightning/tidb-lightning-physical-import-mode.md) 进行数据导入，TiDB Lightning 运行后，TiDB 集群将无法正常对外提供服务。</li><li>如果你不希望 TiDB 集群的对外服务受到影响，可以参考 TiDB Lightning [Logical Import Mode](/tidb-lightning/tidb-lightning-logical-import-mode.md) 中的硬件需求与部署方式进行数据导入。</li></ul> |
+
+## [Backup & Restore (BR)](/br/backup-and-restore-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">通过对大数据量的 TiDB 集群进行数据备份和恢复，实现数据迁移。</span> |
+|---|---|
+| **上游** | TiDB |
+| **下游（或输出文件）** | SST，backup.meta 文件，backup.lock 文件 |
```
Comment (qiancai) [inline migration-tools.md:44]:
```suggestion
| **下游（输出文件）** | SST，backup.meta 文件，backup.lock 文件 |
```

File: migration-tools.md [hunk 12]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+| **下游** | TiDB，MySQL，Kafka，Confluent |
+| **主要优势** | 提供开放数据协议 (TiCDC Open Protocol)。|
+| **使用限制** | TiCDC 只能同步至少存在一个有效索引的表。暂不支持以下场景：<ul><li>单独使用 RawKV 的 TiKV 集群。</li><li>在 TiDB 中创建 SEQUENCE 的 DDL 操作和 SEQUENCE 函数。</li></ul> |
+
+## [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于 TiDB 集群间的增量数据同步，如将其中一个 TiDB 集群作为另一个 TiDB 集群的从集群。</span> |
+|---|---|
+| **上游** | TiDB |
+| **下游（或输出文件）** | TiDB，MySQL，Kafka，增量备份文件 |
```
Comment (qiancai) [inline migration-tools.md:62]:
```suggestion
| **下游（输出文件）** | TiDB，MySQL，Kafka，增量备份文件 |
```

File: migration-tools.md [hunk 13]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+| **主要优势** | <ul><li>支持实时备份和恢复。</li><li>备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复。</li></ul> |
+| **使用限制** | 与部分 TiDB 版本不兼容，不能一起使用。推荐使用 [TiCDC](/ticdc/ticdc-overview.md) 替代 TiDB Binlog。|
```
Comment (qiancai) [inline migration-tools.md:64]:
```suggestion
| **使用限制** | 与部分 TiDB 版本不兼容。推荐使用 [TiCDC](/ticdc/ticdc-overview.md) 替代 TiDB Binlog。|
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Review by Oreoxmt — APPROVED** [review]

File: migration-tools.md [hunk 14]
```diff
@@ -7,35 +7,82 @@ summary: 介绍 TiDB 的数据迁移工具。
...
+| **使用限制** | 数据导入速度与 TiDB Lightning 的 [Logical Import Mode](/tidb-lightning/tidb-lightning-logical-import-mode.md) 大致相同，而比TiDB Lightning 的 [Physical Import Mode](/tidb-lightning/tidb-lightning-physical-import-mode.md) 低很多。建议用于 1 TB 以内的存量数据迁移。 |
+
+## [Dumpling](/dumpling-overview.md)
+
+| 使用场景 | <span style="font-weight:normal">用于将数据从 MySQL/TiDB 进行全量导出</span> |
+|---|---|
+| **上游** | MySQL，TiDB |
+| **下游（输出文件）** | SQL，CSV |
+| **主要优势** | <ul><li>支持全新的 table-filter，筛选数据更加方便</li><li>支持导出到 Amazon S3 云盘</li></ul> |
+| **使用限制** | <ul><li>如果导出后计划往非 TiDB 的数据库恢复，建议使用 Dumpling。</li><li>如果导出后计划往另一个 TiDB 恢复，建议使用 [BR](/br/backup-and-restore-tool.md)。 </li></ul> |
```
Comment (ran-huang) [inline migration-tools.md:28]:
```suggestion
| **使用限制** | <ul><li>如果导出后计划往非 TiDB 的数据库恢复，建议使用 Dumpling。</li><li>如果导出后计划往另一个 TiDB 恢复，建议使用 [BR](/br/backup-and-restore-overview.md)。 </li></ul> |
```

**Pull request merged**